utils.py

```
@@ -0,0 +1,10 @@
+def get_app_version() -> str:
+    return "0.1.0"
+
+
+def generate_token() -> str:
+    return str(random.random())
```

Ruff (0.12.2)

Undefined name `random`

(F821)